NewForce-at-Mountwest/bangazon-workforce-rat
# Jw employees

# Description

When the employee clicks on the Employees item in the navigation bar
Then all current employees should be listed with the following information
First name 
last name
Department


## Related Ticket(s)
TICKET #1

## Steps to Test
Click green Run button to run your app
Given you want to view employees
Click on the Employees link in the navigation bar
All current employees should be listed
And the following information should be presented:
-Employee first name
-Employee last name
-Department Name


# Checklist:
- [x ] I have commented my code, particularly in hard-to-understand areas
- [ x] I have made corresponding changes to the documentation
- [x ] My changes generate no new errors
